src/sentry/integrations/jira/integration.py

```diff
@@ -1039,6 +1042,13 @@ def migrate_issues(self):
             }
         )
 
+    def parse_jira_issue_metadata(self, meta: dict[str, Any]) -> list[JiraIssueTypeMetadata] | None:
+        try:
+            return JiraIssueTypeMetadata.from_jira_meta_config(meta)
+        except Exception as e:
+            sentry_sdk.capture_exception(e)
+            return None
```

Comment:
Eventually this would throw integration exception right?

Reply:
Yes, once we're actively using the model in our code, this will raise if it fails to parse the jira meta config.